conf/client.conf

```diff
@@ -152,6 +152,14 @@ global.metricDummyServerStartPort=9000
 # 是否关闭健康检查: true/关闭 false/不关闭
 global.turnOffHealthCheck=true
 
+# minimal open file limit
```

Comment:
minimal limit? has maximum limit?

Reply:
This minimal limit affects how many TCP sockets the client can create, if it's too low, client can't create TCP sockets which will result in an IO Error.

A safe value for this is twice the number of cluster disks